protocols/v2/subprotocols/mining/src/lib.rs

```diff
@@ -314,20 +357,19 @@ impl ExtendedExtranonce {
         }
     }
 
-    pub fn from_extranonce(
+    // A Extranonce type (in big andian) can be converted into a Extended extranonce type (in big
+    // andian) if ranges are given in input.
```

Comment:
typo.  use the imeperative, you are talking about what this functions does so use something like: It convert an Extranonce and 3 ranges into an ExtendedExtranonce. 

Also is redundant to say "Extranonce type" if you use Extranonce the word "type" is implicit